cmd/up/up.go

```diff
@@ -295,6 +297,27 @@ func Up() *cobra.Command {
 				up.Dev.Autocreate = true
 			}
 
+			// only if the context is an okteto one, we should verify if the namespace has to be woken up
+			if okteto.Context().IsOkteto {
+				// We execute it in a goroutine to not impact the command performance
+				go func() {
+					k8sClient, _, err := okteto.NewK8sClientProvider().Provide(okteto.Context().Cfg)
```

Comment:
We already have it initialised at line 241. Could we reuse that one?

Reply:
Yes, changed to use that one